src/retrace/retrace_worker.py

```diff
@@ -561,9 +561,8 @@ def start_vmcore(self, custom_kernelver=None):
 
             log_debug("Determined kernel version: %s" % kernelver)
 
-        kernelver_str = kernelver.kernelver_str
-
         task.set_kernelver(str(kernelver))
+        kernelver_str = kernelver.kernelver_str
```

Comment:
Why the changed order?

Reply:
It is a minor change which moves the setting of kernelver a bit earlier and goes with the theme of the patch "Set kernelver and vmlinux as soon as possible"